### PR TITLE
passport

### PR DESCRIPTION
**Pull request by katherinejhwang**

### CODE REVIEW

File: client/src/components/Login/Login.js [hunk 1]
```diff
@@ -40,24 +43,66 @@ handleInputChange = event => {
...
     // this.setState({
     //   email: "",
     //   password: ""
     // });
-  };
+	};
+	
+	handleRegisterSubmit = event => {
+    // Preventing the default behavior of the form submit (which is to refresh the page)
+    event.preventDefault();
```
Comment (zanethomas) [inline client/src/components/Login/Login.js]: looks like you have this twice? once at line 48

File: client/src/components/Login/Login.js [hunk 2]
```diff
@@ -40,24 +43,66 @@ handleInputChange = event => {
...
+			headers: {
+				'Accept': 'application/json',
+				'Content-type': 'application/json'
+			},
+			body: JSON.stringify({
+				email: this.state.registeremail,
+				password: this.state.password
+			})
+		})
+		// .then(res => {
```
Comment (zanethomas) [inline client/src/components/Login/Login.js]: is the response meaningful here?

File: client/src/components/Login/Login.js [hunk 3]
```diff
@@ -73,31 +118,31 @@ handleInputChange = event => {
 										<div className="form-group">
 											<label for="email">Email</label>
 											<input
-												type="email"
-												id="email"
-												name="email"
+												type="loginemail"
+												id="loginemail"
```
Comment (zanethomas) [inline client/src/components/Login/Login.js]: loginemail is not a valid type, see: https://www.w3schools.com/html/html_form_input_types.asp

File: client/src/components/Login/Login.js [hunk 4]
```diff
@@ -73,31 +118,31 @@ handleInputChange = event => {
...
 											</input>
 										</div>
 
 										<div className="form-group">
 											<label for="password">Password</label>
 											<input
-												type="password"
-												id="password"
-												name="password"
+												type="loginpassword"
```
Comment (zanethomas) [inline client/src/components/Login/Login.js]: ditto

File: client/src/components/Login/Login.js [hunk 5]
```diff
@@ -117,13 +162,14 @@ handleInputChange = event => {
 							<div className="form-group">
 								<label for="email">Email</label>
 								<input
-									type="email"
-									id="email"
-									name="email"
+									type="registeremail"
```
Comment (zanethomas) [inline client/src/components/Login/Login.js]: etc

File: client/src/components/Login/Login.js (hunk 1 above)
Comment (gglinoga) [reply client/src/components/Login/Login.js]: not sure which is there twice?

Comment (zanethomas) [reply client/src/components/Login/Login.js]: event.preventDefault .... not a big deal, but I think the comments about input type need consideration

File: client/src/components/Login/Login.js (hunk 2 above)
Comment (katherinejhwang) [reply client/src/components/Login/Login.js]: we edited accordingly

File: client/src/components/Login/Login.js (hunk 3 above)
Comment (katherinejhwang) [reply client/src/components/Login/Login.js]: we changed the types

File: client/src/components/Login/Login.js (hunk 4 above)
Comment (katherinejhwang) [reply client/src/components/Login/Login.js]: ditto

**Pull request merged**